File: data/EXP01/python/2300/2300.output_2.py
# @lc app=leetcode id=2300 lang=python3
class Solution:
    def successfulPairs(self, spells, potions, success):
        max_spell = max(spells)
        counter = [0] * (max_spell + 1)
        
        for potion in potions:
            for spell in spells:
                if potion * spell >= success:
                    counter[spell] += 1
                    break
        
        return counter

File: data/EXP01/golang_no_system_var/2300/2300.output_2.go
func successfulPairs(spells []int, potions []int, success int64) []int {
    n := len(spells)
    m := len(potions)
    pairs := make([]int, n)

    for i := 0; i < n; i++ {
        spell := spells[i]
        for j := 0; j < m; j++ {
            potion := potions[j]
            if int64(spell)*int64(potion) >= success {
                pairs[i]++
            }
        }
    }

    return pairs
}
